yaptide/admin/git_submodules.py

```diff
@@ -0,0 +1,44 @@
+import subprocess
+import shutil
+import logging
+import os
+
+
+def find_git_executable():
+    """Function returns git executable path or None"""
+    git_path = shutil.which('git')
+    return git_path
+
+
+def check_submodules_folders_exist():
+    """Function checking if submodules folders exists on system"""
+    with open('.gitmodules') as f:
+        logging.info("Found .gitmodules file")
+        for line in f:
+            if 'path' in line:
+                logging.info("Found submodule line in .gitmodules: %s", line)
+                submodule_path = line.split('=')[-1].strip()
+
+                # Check if folder exists and is not empty
+                if not os.path.exists(submodule_path) or not os.listdir(submodule_path):
```

Comment:
Convert that approach to pathlib, we should use it consistently through all the code